tests/utils/utils.go

```diff
@@ -404,8 +405,21 @@ func CreateNativeTransferTransaction(
 	return SignTransaction(tx, fromKey, subnetInfo.ChainIDInt)
 }
 
+func SendAndWaitForTransaction(ctx context.Context, tx *types.Transaction, subnetInfo SubnetTestInfo) *types.Receipt {
+	err := subnetInfo.ChainWSClient.SendTransaction(ctx, tx)
+	Expect(err).Should(BeNil())
+
+	return WaitForTransaction(ctx, tx.Hash(), subnetInfo)
+}
+
+func WaitForTransactionSuccess(ctx context.Context, txHash common.Hash, subnetInfo SubnetTestInfo) *types.Receipt {
+	receipt := WaitForTransaction(ctx, txHash, subnetInfo)
+	Expect(receipt.Status).Should(Equal(types.ReceiptStatusSuccessful))
+	return receipt
+}
+
 func WaitForTransaction(ctx context.Context, txHash common.Hash, subnetInfo SubnetTestInfo) *types.Receipt {
-	cctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+	cctx, cancel := context.WithTimeout(ctx, 20*time.Second)
```

Comment:
Are these extended timeout/sleeps necessary? If so, can you add a comment explaining why?

Reply:
Nah, just an artifact of merging together what I did and what Cam did independently. Will revert